spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -351,6 +351,15 @@ def assert_flavors
       end
     end
 
+    def assert_flavor_tenant_mapping
```

Comment:
would be nice to have a VCR test that will have the flavor membership set, so you can see there is e.g. flavor accessible only from one tenant